README.md

```diff
@@ -119,9 +119,9 @@ Whether to process user total stats. Set to `0` to disable processing.
 
 Default is unset (processing enabled).
 
-### REALTIME_DIFFICULTY
+### ALWAYS_REALTIME_DIFFICULTY
```

Comment:
How about we call this `PREFER_REALTIME_DIFFICULTY`? I think it makes better grammatical sense.